src/components/widgets/sales-opportunities-funnel/sales-opportunities-funnel.directive.coffee

```diff
@@ -34,13 +33,17 @@ module.controller('WidgetSalesOpportunitiesFunnelCtrl', ($scope, $q, ChartFormat
   w.initContext = ->
     if $scope.isDataFound = angular.isDefined(w.content) && !_.isEmpty(w.content.opps_per_sales_stage) && hasOneOpportunity(w.content.opps_per_sales_stage)
 
-      $scope.statusOptions = _.compact _.map w.metadata.sales_stage_selection, (status) ->
```

Comment:
Yes, by all means. Let's keep `sales_stage_selection` for now: the backend update will be deployed next week.